hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -28,6 +28,9 @@ bittrex_secret_key: null
 blocktane_api_key: null
 blocktane_api_secret: null
 
+btc_markets_api_key: null
+btc_markets_secret_key: null
+
```

Comment:
When changing a `_TEMPLATE.yml` file always remember to add 1 to its version number (`template_version`)